index.html

```diff
@@ -29,13 +32,144 @@ <h2 class="occupation">I’am a Software Developer</h2>
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+            <div class="grid-item">
+                <h3 class="professional-btn">Profesional Art Printing Data</h3>
+                <p class="multiple-p">A daily selection of privately <br> personalized reads; no accounts or
+                    <br>sign-ups required.
+                    has been the industry's standard</p>
+                <ul class="languages">
+                    <li><button class="btn">html</button></li>
+                    <li><button class="btn boot">bootstrap</button></li>
+                    <li><button class="btn">Ruby</button></li>
+                </ul>
+                <button class="see pro">See Project</button>
+            </div>
+
+    </section>
+    <section class="about-me">
+        <h1 class="about">About me</h1>
+        <p class="services">Hello I’m a software developer! I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you see and have a project you need
+            coded, don’t hestiate to contact me.</p>
+        <button class="btnresume">Get My Resume</button>
+        <img src="Images/illustration-about-me.png" class="img-me">
+
```

Comment:
- [ ] Good work so far by implementing the design similar to the design provided by the Figma template. Make sure that you added the image illustration for the About page as a background image and not in the `img` tag. Please, let's rocking, fix it 😄

Reply:
Thank you for this suggestion. I learnt something new by implementing this!